src/app/projects/projlist-filters/projlist-filters.component.scss

```diff
@@ -1,4 +1,16 @@
 @import "~assets/styles/base/base.scss";
+// @import '~@angular/material/theming';
+
+// $custom-theme: mat-bcgov-theme(
+//   (
+//     color: (
+//       primary: '000000',
+//       accent: 'ffd740',
+//       warn: 'ffd740',
+//     ),
+//   )
+// );
+// @include angular-material-theme($custom-theme);
```

Comment:
I'm going to just leave comments for myself to resolve some issues since Jareth is away